core/src/main/java/org/mule/DefaultMuleContext.java

```diff
@@ -323,6 +329,73 @@ public synchronized void start() throws MuleException
         }
     }
 
+    private void startMessageSources() throws LifecycleException
+    {
+        startPipelineMessageSources();
+        startServicesMessageSources();
+    }
+
+    private void startServicesMessageSources() throws LifecycleException
+    {
+        for (Service service : this.getRegistry().lookupObjects(Service.class))
+        {
+            if (service.isStarted() || service.isPaused())
```

Comment:
This is just to maintain the original behaviour. Some objects are created when the source is started only, there are tests that are broken otherwise.